<a href="https://colab.research.google.com/github/cristianramirezrodriguez/practicealgebra1/blob/main/PracticeAssignmentOneVariableEquations.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

**A Practice Assignment To Learn How To Use Python To Solve For x**

*The Solution Set Will Be On Github as SoutionsOneVariableEquations.ipynb*

For my students from EBIA, this is Tech Worksheet 2 Part B

Name:

Any Collaborator Names:

# We have practiced extensively on solving for x when presented a one-variable equation. Our goal today is to use technology to help our process.

Our first step is to move all terms to the left-hand-side of the equation then inputting the left-hand expression into the place where it says "expression" in our python solver.

# The syntax to find the x value of a given one-variable equation is as follows:

from sympy.solvers import solve

from sympy import Symbol

x = Symbol('x')

solve(expression, x)


##For example

## Say you had the equation $x+3 = 5$

## We will first move all terms to the left hand side, producing $x+3-5=0$.

##We can then input the expression on the left-hand side in the solver and run the code below (which produced the correct answer, x=2).

In [1]:
from sympy.solvers import solve
from sympy import Symbol
x = Symbol('x')
solve(x + 3 - 5, x)

[2]

Another example:

Say you want to find $42.1 - x = 2 + x$

You would first move everything over to the left hand side, producing

$42.1-x-2-x=0$

and then input the lefthand expression: $42.1-x-2-x$ into the solver as seen below:

In [2]:
from sympy.solvers import solve
from sympy import Symbol
x = Symbol('x')
solve(42.1-x-2-x, x)


[20.0500000000000]

This produces 20.05, which is the correct answer.

# For each of the following, find x:

1. $2=7x$

2. $-4 = 6x$

3. $1=x+1$

4. $2 = 2x + 22$

5. $12.77 = 33x -2$

6. $9x = 25.9 - 3$

7. $3543x=43-x$

8. $45x + 44 = -35x +2$

9. $2-5=10(x-4)$

10. $4x+5y=0$

11. $y = mx+b$


In [ ]:
#Do number 1 here

In [ ]:
#Do number 2 here

In [ ]:
#Do number 3 here

In [ ]:
#Do number 4 here

In [ ]:
#Do number 5 here

In [ ]:
#Do number 6 here

In [ ]:
#Do number 7 here

In [ ]:
#Do number 8 here

In [ ]:
#Do number 9 here

In [ ]:
#Do number 10 here

In [ ]:
#Do number 11 here